dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -683,15 +683,15 @@ BEGIN
     PUSHBUTTON "S&chimbare pictogramă…", IDC_EDITTYPE_CHANGE_ICON, 150, 10, 70, 14
     LTEXT "&Acțiuni:", -1, 5, 37, 70, 10
     LISTBOX IDC_EDITTYPE_LISTBOX, 5, 50, 140, 60, LBS_NOINTEGRALHEIGHT | LBS_HASSTRINGS | LBS_OWNERDRAWFIXED | LBS_NOTIFY | WS_VSCROLL | WS_TABSTOP
-    PUSHBUTTON "N&ou…", IDC_EDITTYPE_NEW, 150, 50, 70, 14
+    PUSHBUTTON "No&u", IDC_EDITTYPE_NEW, 150, 50, 70, 14
     PUSHBUTTON "&Editare…", IDC_EDITTYPE_EDIT_BUTTON, 150, 65, 70, 14
-    PUSHBUTTON "Eli&mină", IDC_EDITTYPE_REMOVE, 150, 80, 70, 14
+    PUSHBUTTON "Eli&minare", IDC_EDITTYPE_REMOVE, 150, 80, 70, 14
     PUSHBUTTON "&Implicit", IDC_EDITTYPE_SET_DEFAULT, 150, 95, 70, 14
-    AUTOCHECKBOX "Consimte &deschidere după descărcare", IDC_EDITTYPE_CONFIRM_OPEN, 5, 115, 150, 14
-    AUTOCHECKBOX "Afișează e&xtensiea întotdeauna", IDC_EDITTYPE_SHOW_EXT, 5, 130, 150, 14
-    AUTOCHECKBOX "Ca&ută în aceeași fereastră", IDC_EDITTYPE_SAME_WINDOW, 5, 145, 150, 14
-    DEFPUSHBUTTON "Con&firmă", IDOK, 95, 170, 60, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 160, 170, 60, 14
+    AUTOCHECKBOX "Confirmare &deschiderere după descărcare", IDC_EDITTYPE_CONFIRM_OPEN, 5, 115, 150, 14
+    AUTOCHECKBOX "Se afișează întotdeauna extensia", IDC_EDITTYPE_SHOW_EXT, 5, 130, 150, 14
```

Comment:
I am pretty sure, that this autoscheckbox should also have an accelerator. Right?